varnish/test_varnish.py

```diff
@@ -91,7 +91,7 @@ class VarnishCheckTest(AgentCheckTest):
 
     def _get_varnish_stat_path(self):
         varnish_version = os.environ.get('FLAVOR_VERSION', VARNISH_DEFAULT_VERSION).split('.', 1)[0]
-        return "%s/ci/varnishstat%s" % (os.path.dirname(os.path.abspath(__file__)), varnish_version)
+        return ["%s/ci/varnishstat%s" % (os.path.dirname(os.path.abspath(__file__)), varnish_version)]
```

Comment:
This breaks other stuff. You should instead make the path into a list in the `self.check._get_version_info(self._get_varnish_stat_path())` call on line 132